[April CQA] CQA fixes for the UI title (terminology, UI formatting, style cleanup) (#361)
migtools/mta-documentation: docs/topics/mta-ui/proc_defining-an-archetype.adoc

= Creating an archetype

[role="_abstract"] 

> To assess several applications with shared characteristics simultaneously, define an archetype in the {ProductFullName} user interface (UI).

Author: ```suggestion
To assess a group of applications with shared characteristics, define an archetype in the {ProductFullName} user interface (UI).
```
